packages/core/jest.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    moduleNameMapper: {
```

Comment:
what is this used for?

Reply:
It is used to allow alias imports to work. It doesn't automatically pick up webpack aliases, so we have to add this for each package and extend the root config.